src/anchor-navigation/scroll-spy.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
_Almost_ every file in the repo that contains only a hook has the `use-` prefix in the file name. I'd suggest to follow this pattern here as well.